### DISCUSSION TITLE
Congrats! Your request for @Ayesha-batool34 was approved and your benefits will be available soon.

### DISCUSSION THREAD

**Discussion by muhammadshoaibali123** [community/community#130885] [GitHub Education]
### Hi everyone,

Congrats! Your request for [@Ayesha-batool34](https://github.com/Ayesha-batool34) was approved and your benefits will be available [soon]

When and how i can access my Academic benefits?

**Comment by davevad93** [comment]
Hi @Ayesha-batool34 ,

It usually takes only a few days for benefits to become available but sometimes it can take longer, the waiting time may vary depending on various factors, such as the volume of requests.

When the benefits are available the status will change to:

`Congrats! Your request for @Ayesha-batool34 was approved and your benefits are now available`
